client/web/compose/src/components/ModuleFields/Editor/Geometry.vue

```diff
@@ -156,56 +156,30 @@
         {{ $t('clickToPlaceMarker') }}
       </template>
 
-      <div
-        v-if="!field.options.hideGeoSearch"
-        class="geosearch-container"
-      >
-        <c-input-search
-          v-model="geoSearch.query"
-          :placeholder="$t('geosearchInputPlaceholder')"
-          :autocomplete="'off'"
-          :debounce="300"
-          @input="onGeoSearch"
-        />
-
-        <div class="geosearch-results">
-          <div
-            v-for="(result, idx) in geoSearch.results"
-            :key="idx"
-            class="geosearch-result"
-            @click="placeGeoSearchMarker(result)"
-          >
-            {{ result.label }}
-          </div>
-        </div>
-      </div>
-
       <c-map
         ref="c-map"
         :field="field"
         :map="map"
-        :on-location-found="onLocationFound"
-        :hide-current-location-button="!field.options.hideCurrentLocationButton"
-        :local-value-index="localValueIndex"
+        :hide-geo-search="!field.options.hideGeoSearch"
```

Comment:
No need to negate the value here, otherwise its gonna be the opposite of what you want to do